v7.3.3 release is missing `dist` directory

protobuf.js version: 7.3.3 (also occurred in v7.3.1)

The latest release is missing the `dist` directory when installed via npm. 

```
✘ [ERROR] Could not resolve "protobufjs/dist/minimal/protobuf.js"

    packages/engine/Source/Core/GoogleEarthEnterpriseMetadata.js:1:26:
      1 │ import * as protobuf from "protobufjs/dist/minimal/protobuf.js";
        ╵                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Is this purposeful? If it is, we'll update to the new path. But it does not appear in the changelog, and the `dist/` path is still listed in `package.json` under `files`.

See https://github.com/CesiumGS/cesium/issues/12143

--- a/package.json
+++ b/package.json
@@ -40,6 +40,7 @@
     "lint:types": "tslint \"**/*.d.ts\" -e \"**/node_modules/**\" -t stylish -c config/tslint.json",
     "pages": "node scripts/pages",
     "prepublish": "cd cli && npm install && cd .. && npm run build",
+    "prepublishOnly": "cd cli && npm install && cd .. && npm run build",
     "postinstall": "node scripts/postinstall",
     "prof": "node bench/prof",
     "test": "npm run test:sources && npm run test:types",

